freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClientExceptionUtil.java

```diff
@@ -157,6 +157,22 @@ public static void ignoreNotFoundException(FreeIpaClientRunnable runnable, Strin
         }
     }
 
+    public static <T> Optional<T> ignoreEmptyModExceptionWithValue(FreeIpaClientCallable<T> callable, String message, Object... messageParams)
+            throws FreeIpaClientException {
+        try {
+            return Optional.ofNullable(callable.run());
+        } catch (FreeIpaClientException e) {
+            if (isEmptyModlistException(e)) {
+                Optional.ofNullable(message).ifPresentOrElse(
```

Comment:
Just curious. 
Is it the same to write 

if (message != null) {
} else {
}

Reply:
yes it is the same, just a compact format